services/ffc/ffc-demo/base/patch/kustomize.yaml

```diff
@@ -26,6 +26,7 @@ spec:
       SERVICE_CPU_QUOTA: "3000"
```

Comment:
This one is also not having correct variables..

Reply:
Proper variables are updated